conf/messages.en

```diff
@@ -47,7 +47,7 @@ date.month.11 = November
 date.month.12 = December
 
 timeout.title = You’re about to be signed out
-timeout.message = For security reasons, you will be signed out of this service in
+timeout.message = For security reasons, you will be signed out in
```

Comment:
The copy-deck has the content `For your security, we will sign you out in` (https://docs.google.com/spreadsheets/d/1og4lluJ0o75kvX5j2Ma_W54eUf_OAl6Ud8EfMYP14zI/edit?pli=1#gid=868164043)